client/main.js

```diff
@@ -0,0 +1,470 @@
+import BN from "bn.js";
+import web3Utils from "web3-utils";
+
+const ganache = require("ganache-core");
+const contract = require("truffle-contract");
+const ReputationMiningCycleJSON = require("../build/contracts/ReputationMiningCycle.json");
+const ColonyNetworkJSON = require("../build/contracts/IColonyNetwork.json");
+const PatriciaTreeJSON = require("../build/contracts/PatriciaTree.json");
+
+const ReputationMiningCycle = contract(ReputationMiningCycleJSON);
+
+const jsonfile = require("jsonfile");
+
+const file = "./reputations.json";
+
+const Web3 = require("web3");
+
+const ColonyNetwork = contract(ColonyNetworkJSON);
+const accountAddress = "0xbb46703786c2049d4d6dd43f5b4edf52a20fefe4";
```

Comment:
How is this different from `this.minerAddress`?

Reply:
Why is this hardcoded? I think there's room for improvement. Again, we might be able to use the tools we built for the colony-js client

Reply:
This hardcoded address is an address used by the mining client locally on its own instance of `ganache`, which it uses to build the PatriciaTree. The `secretKey` is the corresponding secret key for this address.  This key is never used on the public chain, so all the mining clients having the same one isn't an issue.

The `minerAddress` is the address used by a mining client interacting with the public chain; that address should be the address they are in control of that is staking `CLNY` to be able to submit new reputation hashes during the mining process. Those obviously should be different between different clients!